external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/internal/OffsetManager.java

```diff
@@ -118,6 +118,8 @@ public OffsetAndMetadata findNextCommitOffset() {
 
         OffsetAndMetadata nextCommitOffsetAndMetadata = null;
         if (nextCommitMsg != null) {
+            //The committed offset should be the next message
+            nextCommitOffset = nextCommitOffset + 1;
```

Comment:
I would prefer that we fix the offsets in the code block above (L76-117) instead of adjusting the offset here. That way the log messages won't be printing the wrong offset.